### ISSUE TITLE
Add an option to sort by latest added products

### ISSUE THREAD

**Issue by bitcoinuser** [OpenBazaar/openbazaar-desktop#897]
Brief Description: On Openbazaar I see that I can only sort by relevance, price and rating. I would like to sort by latest added products

OpenBazaar version (shown on `About OpenBazaar` page in menu):
Client 2.0.11, server 0.9.1

**Comment by rmisio** [comment]
this is a duplicate of #217, which is waiting on a [server issue](https://github.com/OpenBazaar/openbazaar-go/issues/276) to be resolved.
